EventFilter/L1TRawToDigi/plugins/implementations_stage2/EMTFCollections.cc

```diff
@@ -30,13 +30,22 @@ namespace l1t {
           EMTFCPPFs_ZS_->push_back(c);
         }
       }
+      // TODO: Do we need something equivalent for GEMs - JS 03.07.20
+      // for (const auto& g : *EMTFGEMPadClusters_) {
```

Comment:
Please remove this commented-out code.  It looks like it is a copy of some other code from elsewhere?  Is there going to be another PR which will implement an analogous code here ?

Reply:
Indeed, if one expands the diff, the previous section has exactly this block for CPPF, and this comment is specifically to ensure that it's not forgotten by the developer to check whether such similar logic is *also* needed for the GEM digis.